docs/component-audits/wave-35-interaction-evidence-report.md

```
@@ -0,0 +1,71 @@
+# Wave 3.5 — Interaction Evidence Hardening
+
+Baseline: `f5d7f3279565ea2136f75018f24f7ca4318d983d` (merged PR
+#17). Branch: `chore/interaction-evidence-hardening`. This is a narrow
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)